### PR TITLE
Add Time Machine

### PR DESCRIPTION
**Pull request by kimg45**
Added Time Machine to the backups section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1342,6 +1342,12 @@ Ensure all eligible online accounts have [multi-factor authentication](https://e
 
 Encrypt files locally before backing them up to external media or online services.
 
+You should follow the [3-2-1 backup model](https://www.cisa.gov/sites/default/files/publications/data_backup_options.pdf) as outlined by CISA.
```
Comment (drduh) [inline README.md:1345]: Let's not have this just become a catalog of links, especially to PDFs; explain terminology as it's defined.

File: README.md [hunk 2]
```diff
@@ -1342,6 +1342,12 @@ Ensure all eligible online accounts have [multi-factor authentication](https://e
...
+
+[Time Machine](https://support.apple.com/104984) is the built-in tool for handling backups on macOS. Get an external drive or network drive to back up to and [encrypt](https://support.apple.com/guide/mac-help/keep-your-time-machine-backup-disk-secure-mh21241) your backups.
+
+You can also do backups manually if you want.
```
Comment (drduh) [inline README.md:1349]: This advice is not really actionable, remove it or justify.

**Review by drduh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kimg45) [reply README.md]: added an explanation

File: README.md (hunk 2 above)
Comment (kimg45) [reply README.md]: removed

**Pull request merged**